File: Scripts/laptop/palindrome-detection/php/palindrome-detection.php
<?php
function is_palindrome($string) {
  return $string == strrev($string);
}

$r = 1;

function executeTask($i) {
	$result = is_palindrome("saippuakivikauppias" . $i);
	return $i;
}

for ( $i = 0; $i < 2000000000; ++$i ) {
	$r = executeTask($i);
}
?>

File: Scripts/laptop/palindrome-detection/python/palindrome-detection.py
def is_palindrome(s):
  return s == s[::-1]

r = 1
def executeTask(i):
	b = is_palindrome("saippuakivikauppias")
	return i + 1

for i in xrange(2000000000):
	r = executeTask(i + r)
